modules/serverless-overriding-pdbs-eventing.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/tuning/serverless-ha.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-eventing_{context}"]
+= Overriding disruption budgets
+
+A Pod Disruption Budget (PDB) is a standard feature of Kubernetes APIs that helps limit the disruption to an application when its pods need to be rescheduled for maintenance reasons.
+
+.Procedure
+
+You can override the default PDB for a specific resource by modifying the `minAvailable` configuration value in the `KnativeEventing CR`.
+
+[NOTE]
+====
+If you disable high-availability, for example, by changing the `high-availability.replicas` value to 1, make sure you also update the corresponding podDisruptionBudget `minAvailable` value to 0. Otherwise, the podDisruptionBudget will prevent automatic cluster or operator upgrades.
```

Comment:
```suggestion
If you disable high-availability, for example, by changing the `high-availability.replicas` value to 1, make sure you also update the corresponding pod disruption budget `minAvailable` value to 0. Otherwise, the pod disruption budget prevents automatic cluster or operator upgrades.
```

Reply:
@ochromy Did you mean to leave the first suggestion out in this paragraph? (Changing `podDisruptionBudget` to `pod disruption budget`)